examples/visual-tests/amp-story/amp-story-shopping-rtl.html

```diff
@@ -0,0 +1,204 @@
+<!doctype html>
```

Comment:
Please see the previous comment about adding dir=rtl to a parent element.
In this way we can have all examples in one file and we wont need this file.